examples/Civis_WBCD_2017.ipynb

```diff
@@ -0,0 +1,676 @@
+    "\n",
+    "CivisML uses the Civis Platform to train machine learning models and parallelize their predictions over large datasets. It contains best-practice models for general-purpose classification and regression modeling as well as model quality evaluations and visualizations. All CivisML models use scikit-learn for interoperability with other platforms and to allow you to leverage resources in the open-source software community when creating machine learning models. CivisML automates many of the repetitive tasks a Data Scientist performs in scikit-learn, making it possible to try many more modeling ideas with many fewer lines of code. \n",
+    "\n",
+    "In this notebook, I will explore some of my major use cases (e.g. trying different algorithms, firing off many models in unison with different hyperparameters, and analyzing cross-validated results). The classifiers I build below are all from publicly available data at the UCI Machine Learning Repository[1][2]. Learn more about the [dataset](https://archive.ics.uci.edu/ml/machine-learning-databases/breast-cancer-wisconsin/).\n",
+    "\n",
+    "**CivisML uses the Civis API.  If you'd like to run this code yourself, consider signing up for a [free trial](https://www.civisanalytics.com/civis-platform-signup/).**\n",
+    "\n",
+    "---\n",
+    "The Wisconsin Breast Cancer (Diagnostic) Data Set records anonymous data on ~600 patients' cells, along with whether those cells are cancerous or benign. We can create a classifier to predict which cells are dangerous!\n",
+    "\n",
+    "---\n",
+    "[1]: O. L. Mangasarian and W. H. Wolberg: \"Cancer diagnosis via linear programming\", SIAM News, Volume 23, Number 5, September 1990, pp 1 & 18.\n",
+    "\n",
+    "[2]: Lichman, M. (2013). UCI Machine Learning Repository [http://archive.ics.uci.edu/ml]. Irvine, CA: University of California, School of Information and Computer Science."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### TL;DR: Why Would I Use This?\n",
+    "- CivisML easily parallelizes your modeling tasks, making it fast to try lots of ideas\n",
+    "- Diagnostic and other results are accessible in fewer lines of code\n",
+    "- The package works with any scikit-learn estimators, so if you have your own pipeline already you can still get the benefits of CivisML"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# packages for downloading the data\n",
+    "import os\n",
+    "import urllib\n",
+    "\n",
+    "# Packages for munging, plotting, machine learning\n",
+    "import pandas as pd\n",
+    "import xgboost\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "# packages for CivisML\n",
+    "from concurrent import futures\n",
```

Comment:
This is standard library, so I think it doesn't fit in a section titled "packages for CivisML".